workitem/link/link_repository.go

```diff
@@ -458,3 +459,30 @@ func (r *GormWorkItemLinkRepository) WorkItemHasChildren(ctx context.Context, pa
 	}
 	return hasChildren, nil
 }
+
+// GetParentID returns parent ID of the given work item if any
+func (r *GormWorkItemLinkRepository) GetParentID(ctx context.Context, ID uuid.UUID) (*uuid.UUID, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitemlink", "get", "parent"}, time.Now())
+	query := fmt.Sprintf(`
+			SELECT id FROM %[1]s WHERE id in (
+				SELECT source_id FROM %[2]s
+				WHERE target_id = $1 AND deleted_at IS NULL AND link_type_id IN (
+					SELECT id FROM %[3]s WHERE forward_name = 'parent of'
+				)
+			)`,
+		workitem.WorkItemStorage{}.TableName(),
+		WorkItemLink{}.TableName(),
+		WorkItemLinkType{}.TableName())
+	var parentID uuid.UUID
+	db := r.db.CommonDB()
+	stmt, err := db.Prepare(query)
```

Comment:
Why do you create a prepared statement here? You're only using that statement once and close it afterwards. I would agree that a statement is a good idea if it is stored somewhere and maybe reused but not this way.

Reply:
true, done in https://github.com/fabric8-services/fabric8-wit/pull/1592/commits/ba2c6543d052dc8381989222eba692bc9efa42a3